.devcontainer/init_env/config_space.sh

```
@@ -90,7 +90,20 @@ if [ ! -d "node_modules/playwright" ] || [ ! -d "$HOME/.cache/ms-playwright" ];
     sudo npx playwright install-deps
 fi
 
-sudo chmod -R 766 logs
+# Setup proper permissions for logs and tmp directories
+echo "Setting up logs and tmp directories..."
+sudo mkdir -p $(echo $REPO_PATH)/app/logs
```

Shellcheck (0.11.0)

[warning] Quote this to prevent word splitting.

(SC2046)